pkg/messaging/message.go

```diff
@@ -1,6 +1,16 @@
 package messaging
 
-import "encoding/json"
+import (
+	"encoding/json"
+
+	"github.com/montrosesoftware/tarpon/pkg/logging"
+)
+
+const (
+	ServerUID        = "tarpon"
```

Comment:
Can you make it lower case as rest of the constants?

Reply:
I made it upper case because we need to export it and use it [here](https://github.com/montrosesoftware/tarpon/blob/FP-180/pkg/server/server.go#L199), otherwise it wont compile.

Reply:
I see. Good point.